### PR TITLE
Fix Monitor bug Quarkus #7444

### PR DESCRIPTION
**Pull request by Teg79**
Porting here the same fix as per merge https://github.com/quarkusio/quarkus/pull/7445/
It resolves the https://github.com/quarkusio/quarkus/issues/7444

### CODE REVIEW

File: vertx/src/main/java/io/undertow/vertx/VertxHttpExchange.java [hunk 1]
```diff
@@ -19,11 +19,7 @@
 import io.vertx.core.AsyncResult;
 import io.vertx.core.Handler;
 import io.vertx.core.buffer.Buffer;
-import io.vertx.core.http.HttpHeaders;
-import io.vertx.core.http.HttpMethod;
-import io.vertx.core.http.HttpServerRequest;
-import io.vertx.core.http.HttpServerResponse;
-import io.vertx.core.http.HttpVersion;
+import io.vertx.core.http.*;
```
Comment (gsmet) [inline vertx/src/main/java/io/undertow/vertx/VertxHttpExchange.java:22]: We try to avoid star imports, could you fix that?

**Review by gsmet — DISMISSED** [review]
Thanks for this PR! Added a suggestion inline.

**Review by Teg79 — COMMENTED** [review]
Removed * imports

**Comment by gsmet** [comment]
Thanks! I squashed the 2 commits as we use merge commits and we can't squash via GH UI.

**Review by stuartwdouglas — APPROVED** [review]

**Pull request merged**